crates/relayer/src/light_client/axon.rs

```diff
@@ -101,22 +104,22 @@ impl super::LightClient<AxonChain> for LightClient {
         trusted: ibc_relayer_types::Height,
         target: ibc_relayer_types::Height,
         client_state: &AnyClientState,
-    ) -> Result<Verified<<AxonChain as ChainEndpoint>::LightBlock>, Error> {
-        todo!()
+    ) -> Result<Verified<AxonLightBlock>, Error> {
+        Ok(Verified {
+            target: AxonLightBlock::default(),
+            supporting: vec![],
+        })
     }
 
     fn check_misbehaviour(
         &mut self,
         update: &ibc_relayer_types::core::ics02_client::events::UpdateClient,
         client_state: &AnyClientState,
     ) -> Result<Option<MisbehaviourEvidence>, Error> {
-        todo!()
+        Ok(None)
```

Comment:
I mean if `todo!()` is left here, that means the Hermes runtime won't make use of it and can be identified as a useless method so that not to put attention on it